hackathon_site/event/jinja2/event/landing.html

```diff
@@ -234,44 +232,37 @@ <h3 class="center white-text">FAQ</h3>
     </div>
     <br /><br />
 </div>
-<div class="ombreBackgroundDark2">
-    <div class="container section scrollspy" id="sponsors">
-        <div class="row">
-            <h3 class="center white-text">Sponsors</h3>
-            <div class="row sponsorImgs">
-                <!-- Platinum -->
-                <a href="https://www.daisyintelligence.com/" class="col m6 s12" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Daisy_Intelligence_white.svg') }}" alt="Daisy Intelligence"/></a>
-                <a href="https://tractionondemand.com/" class="col m6 s12" rel="noopener" target="_blank"><img src="{{ static('event/images/sponsors/Traction_On_Demand_white.svg') }}" alt="Traction on Demand" /></a>
+<!--<div class="ombreBackgroundDark2">-->
+<!--    <div class="container section scrollspy" id="sponsors">-->
+<!--        <div class="row">-->
+<!--            <h3 class="center white-text">Sponsors</h3>-->
+<!--            <div class="row sponsorImgs">-->
+<!--                &lt;!&ndash; Platinum &ndash;&gt;-->
```

Comment:
Undo the changes on the already-commented lines. No need to render `&lt;!&ndash;`, just keep the line as is because it's already a comment. Same for the lines below: Gold, Silver, Bronze, Partners

Reply:
I see. At first, I just select all and ctrl+/ to comment out the sponsor part.